blamo/lib/Unit/unit.dart

```diff
@@ -282,6 +317,27 @@ class _UnitPageState extends State<UnitPage> {
     );
   }
 
+  Future<bool> checkUnitDepthOverlap() async {
+    ObjectHandler objectHandler = new ObjectHandler();
+    for(int i = 0; i < currentState.unitList.length; i++){
+      Unit currentCheck = await objectHandler.getUnitData(currentState.unitList[i], currentState.currentDocument);
+      if(currentState.currentUnit != currentState.unitList[i]) {
+        if (unitObject.depthUB < currentCheck.depthUB && unitObject.depthUB > currentCheck.depthLB) {
```

Comment:
Error thrown here if trying to save an existing test with modification, prevents the saving.